src/deluge/dsp/filter/svf.h

```diff
@@ -43,6 +43,7 @@ class SVFilter : public Filter<SVFilter> {
 	inline q31_t doSVF(q31_t input, SVFState& state);
 	SVFState l;
 	SVFState r;
+	bool band_mode;
```

Comment:
It's probably not super important here (since our minimum allocation granularity is 4 bytes anyway) but this forces the compiler to add 3 bytes of padding. If it's at the end the size can technically be 3 bytes lower.